specification/cognitiveservices/AnomalyDetector/tspconfig.yaml

```diff
@@ -14,10 +14,11 @@ emit: [
 ]
 options:
   "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "./data-plane"
```

Comment:
@markcowl do you know why this property is needed explicitly? I tried to just hardcode it in the output-file path directly but when I did the output didn't go to the correct place because it seems as though service-name, version, version-status were all null.

Reply:
Yes, it is essentially a toggle switch that is necessary for example inclusion to work correctly with versioning.  We should make it unnecessary.